src/rules/rule-no-duplicate-properties/index.js

```diff
@@ -34,7 +34,7 @@ export default function (actual) {
         }
         if (child.type !== "decl") { return }
         const prop = child.prop
-        if (decls.indexOf(prop) !== -1) {
+        if (prop !== "src" && decls.indexOf(prop) !== -1) {
```

Comment:
Could you add a comment here explaining the justification, just so that if we look at this code a year from now it's evident why this line is in there?